DualView/src/androidTest/java/com/microsoft/device/display/samples/contentcontext/ParcelableTest.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.contentcontext
+
+import android.os.Parcel
+import androidx.test.filters.SmallTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.microsoft.device.display.samples.contentcontext.model.Restaurant
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.hamcrest.CoreMatchers.`is` as iz
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@SmallTest
+class ParcelableTest {
```

Comment:
Everything looks good! if anything to add is here, that I believe you are testing something that we expect the framework has already tested: the parcelable creation. If so, we could get rid of this test, but it's already done and here so it doesn't hurt. Your call Bianca ;-)